frontend/src/lib/components/SmoothingFilter/smoothings.ts

```diff
@@ -4,6 +4,16 @@ import { IntervalType, SmoothingType } from '~/types'
 // the typing should catch if we update SmoothingType but do not add an explicit
 // option to this lookup
 export const smoothingOptions: Record<IntervalType, { label: string; value: SmoothingType }[]> = {
+    minute: [
+        {
+            label: 'No smoothing',
+            value: 1,
+        },
+        {
+            label: '3 minute',
+            value: 3,
```

Comment:
Why 3 and not the more "round" 5?

BTW the labels are all differently styled: "3 minute", "24 Hrs", "7 Day". Let's reformat them to all look like this: "24-hour average", "7-day average"